lzy/channel-manager/src/main/java/ai/lzy/channelmanager/services/ChannelManagerPrivateService.java

```diff
@@ -109,6 +111,11 @@ public void destroy(LCMPS.ChannelDestroyRequest request, StreamObserver<LongRunn
             return;
         }
 
+        var idempotencyKey = IdempotencyUtils.getIdempotencyKey(request);
```

Comment:
this is not enough
we can have race on line 125, and we should handle it

see at https://github.com/lambdazy/lzy/blob/e7aba7d04d943c7b3cde69b23e151c16e3b830c5/lzy/allocator/src/main/java/ai/lzy/allocator/services/AllocatorService.java#L213